style.css

```diff
@@ -1,574 +1,495 @@
-}
+  .head-button {
+    background-color: #1a2236;
+    color: #fff;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    text-align: center;
+    height: 50px;
+    margin-left: 8%;
+    margin-right: 8%;
+  }
 
-.card {
-  width: 100%;
-  height: 344px;
-  display: flex;
-  flex-direction: column;
-}
+  .work {
+    margin-top: 20px;
+  }
 
-.img {
-  width: 100%;
-  background-repeat: no-repeat;
-  height: 187px;
-  border: 0.333333px solid rgba(255, 255, 255, 0.1);
-}
+  .geometry {
+    width: 100%;
+  }
 
-.img-one {
-  background-image: url(./images/card-one.svg);
-}
+  .work h2 {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 50px;
+    margin-top: 20px;
+    margin-bottom: 20px;
+  }
 
-.img-two {
-  background-image: url("./images/card-two.svg");
-}
+  .card img {
+    width: 100%;
+  }
 
-.img-three {
-  background-image: url("./images/card-three.svg");
-}
+  .project {
+    background-color: #1a2236;
+    height: 156px;
+  }
 
-.img-four {
-  background-image: url("./images/card-four.svg");
-}
+  .project p {
+    margin-top: 0;
+    margin-bottom: 0;
+    color: #fff;
+    font-style: normal;
+    font-weight: 300;
+    padding-left: 19px;
+    padding-top: 20px;
+    font-size: 20px;
+  }
 
-.img-five {
-  background-image: url("./images/card-five.svg");
-}
+  .language {
+    padding-left: 10px;
+  }
 
-.img-six {
-  background-image: url("./images/card-six.svg");
-}
+  .language li {
+    display: inline;
+    float: left;
+    list-style: none;
+    margin-left: 10px;
+    background-color: #98dede;
+  }
 
-.project {
-  width: 85%;
-  height: 160px;
-  background: #1a2236;
-  border: 0.333333px solid rgba(255, 255, 255, 0.1);
-  backdrop-filter: blur(0.5px);
-  display: flex;
-  flex-direction: column;
-  padding-left: 20px;
-  padding-top: 20px;
-  gap: 16px;
-}
+  .material-icons {
+    width: 10.57px;
+    height: 15.35px;
+    padding: 0 12px;
+  }
 
-.project > p {
-  width: 100%;
-  height: 16px;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 300;
-  font-size: 20px;
-  line-height: 80%;
-  color: #fff;
-}
+  .btn {
+    display: flex;
+    background-color: #1a2236;
+    color: #fff;
+    width: 190px;
+    height: 50px;
+    margin-top: 50px;
+    margin-left: 10px;
+  }
 
-.language {
-  display: flex;
-  gap: 10px;
-}
+  .btn p {
+    font-weight: 300;
+    font-size: 16px;
+    line-height: 24px;
+    padding-top: 10px;
+    padding-left: 0;
+  }
 
-.language > li {
-  background: #98dede;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 300;
-  font-size: 10.3334px;
-  line-height: 80%;
-  color: #1a2236;
-  padding: 6px 10px;
-}
+  .about-me h2 {
+    width: 50%;
+    height: 50px;
+    font-family: "Playfair Display", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 50px;
+    color: #1a2236;
 
-ul {
-  list-style-type: none;
-}
+    /* margin-top: 10px; */
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly remove all commented code from your project as this is regarded as a bad practice.